include/fcntl.h

```diff
@@ -66,6 +66,7 @@
 #define O_DSYNC     O_SYNC          /* Equivalent to OSYNC in NuttX */
 #define O_BINARY    (1 << 8)        /* Open the file in binary (untranslated) mode. */
 #define O_DIRECT    (1 << 9)        /* Avoid caching, write directly to hardware */
+#define O_CLOEXEC   (1 << 10)       /* Close on execute */
```

Comment:
Are you going to implement support for this flag?  Or is is just to complete a compile.  If it is just for the compile why not set it zero.

That is what is done for the other flags below under "Unsupported, but required open flags"

Reply:
No implementation, just to ensure the compatibility of linux applications

Reply:
The zero would be better value.  This wastes a bit and we may want to reuse for another purpose in the future.

I should be moved to a section like;

    /* Unsupported, but needed for Linux compatibility */
   
    #define O_CLOEXEC   (0)              /* Close on execute */

That makes you intention clearer and does not waste a bit.

Reply:
Got, which seems more appropriate.

Reply:
what's the point to make the app compile? it likely doesn't work as expected. a compile-time error is even better.

Reply:
Because, this flag isn't very critical, some application just add this flag when it call open, but don't care the difference really.

Reply:
Yes, compile error is better but many applications come from other open source projects, in this case, flag O_CLOEXEC does not particularly affect the application behavior in my understanding, the change just make the application to run more smoothly in different OS.

Reply:
it might not be critical for some apps. it might be critical for others. we don't have a priori knowledge. an application for which this flag is not critical can always do #ifdef O_CLOEXEC. providing dummy definition can even break such applications.

Reply:
Ok, I will upload the implementation later, please keep this PR.

Reply:
Change have been updated, Please help to review again, thanks!